e2e/tests/plugin-form-ReadOnlyPrimitives.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { expect, test } from '@playwright/test'
+
+test('Read only primitives', async ({ page }) => {
+  await page.goto('http://localhost:3000/')
+  await page.getByText('plugins', { exact: true }).click()
+  await page.getByText('form').click()
+  await page.getByText('read_only_primitives').click()
+  await page.getByText('ReadOnlyPrimitives').click()
+
+  const checkNotEditable = async () => {
+    await expect(
+      page.getByTestId('stringRequired').getByTestId('form-textfield')
```

Comment:
Perhaps write
```suggestion
      page.getByLabel('A required string')
```
instead?